contracts/notification-contracts/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+server.port=8080
```

Comment:
Don't think the contracts should have these configurations
They're not Spring Boot apps

The notification-service / email-service should have these